apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-description-links/src/inline-support-link.tsx

```diff
@@ -0,0 +1,40 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { ExternalLink } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { useState, JSXElementConstructor, ReactElement } from 'react';
+
+interface Props {
+	children: string | ReactElement< string | JSXElementConstructor< any > >;
+	title: string;
+	url: string;
+}
+export default function InlineSupportLink( { children, title, url }: Props ) {
+	// This was cooked up to only apply the link in the BlockEditor sidebar.
+	// Since there was no identifier in the environment to differentiate.
+	const [ ref, setRef ] = useState< Element | null >();
+
+	if ( ref && ! ref?.closest( '.block-editor-block-inspector' ) ) {
+		return <>{ children }</>;
+	}
+
+	return (
+		<>
+			{ children }
+			<br />
+			<ExternalLink
+				onClick={ () => {
+					recordTracksEvent( 'calypso_block_description_links', {
```

Comment:
From the discussion I was pinged on (p2rHU0-Tk-p2#comment-3127), I don't know if the goal is to track the user clicking on the `InlineSupportLink` itself, or the `ExternalLink` contained there within. This PR tracks the latter, whereas the [`InlineSupportLink` Calypso component](https://github.com/Automattic/wp-calypso/blob/trunk/client/components/inline-support-link/index.jsx) tracks the former. I'll let you decide which action (or both) that you'd like to track.

Following the Tracks naming conventions (`<source>_<context>_<optional subcontext>_<action>_<optional qualifier>`), I think this should end with an action (i.e. `_click`). The goal would be for someone not familiar with the code to be able to look at the event name and understand what the user did to trigger it. Elsewhere in the editing toolkit plugin, I see examples like `calypso_customizer_global_styles_block_editor_link_clicked` and `calypso_editor_sidebar_item_add`. Depending on what you decide you want to track from my first point above, maybe this should be `calypso_block_description_support_link_click` or `calypso_block_description_support_external_link_click`.

Reply:
@michaeldcain I think there may be some overlap in component naming, now that I look a little closer. The `InlineSupportLink` in this package is the custom component to wrap `ExternalLink` with an added ref. To avoid confusion from the Calypso component I changed the name to `DescriptionSupportLink`.

For the tracker event naming, I will update this to fit the naming convention. 👍